src/transformers/modeling_tf_utils.py

```diff
@@ -2213,17 +2213,24 @@ def save_pretrained(
             )
             for shard_file, shard in shards.items():
                 with h5py.File(os.path.join(save_directory, shard_file), mode="w") as shard_file:
-                    save_attributes_to_hdf5_group(
-                        shard_file,
-                        "layer_names",
-                        ["/".join(layer.name.split("/")[1:]).encode("utf8") for layer in shard],
-                    )
-
+                    layers = []
```

Comment:
this seems like a wrong commit no? Don't think we should touch `tf` here

Reply:
Wrong commit really sorry

Reply:
Reverted